seed/data/ref/00200_reduction_reason.js

```diff
@@ -1,5 +1,5 @@
 var tableName = 'reduction_reason'
-var insertStatement = 'INSERT INTO app.' + tableName + ' (id, reason, reason_short_name, category_id, allowance_percentage, max_allowance_percentage, months_to_expiry, fixed_allowance_hours) VALUES '
```

Comment:
I take it this only needed for applying the hours value for CMS? I wasn't sure!

Reply:
Yeh, the reductions ref data uses percentage rather than absolute hours